node/external/transactionAPI/apiTransactionResults.go

```diff
@@ -111,23 +116,21 @@ func (arp *apiTransactionResultsProcessor) putSmartContractResultsInTransactionB
 }
 
 func (arp *apiTransactionResultsProcessor) putLogsInTransaction(hash []byte, tx *transaction.ApiTransactionResult, epoch uint32) {
```

Comment:
Let's add a TODO here and in the following functions:
the name starts with `put xxx` and does a "get" and a **set field**. This is a bit misleading, in my opinion. 
No good suggestion at this moment how to refactor it.

Reply:
Renamed to `loadLogsIntoTransaction`.